solr/solr-ref-guide/src/config-sets.adoc

```diff
@@ -76,4 +82,16 @@ curl -v -X POST -H 'Content-type: application/json' -d '{
 
 == Configsets in SolrCloud Mode
 
-In SolrCloud mode, you can use the <<configsets-api.adoc#configsets-api,Configsets API>> to manage your configsets.
+In SolrCloud, it's critical to understand that configsets are fundamentally stored in ZooKeeper _and not_ the file system.
+Solr's `_default` configset is uploaded to ZooKeeper on initialization.
+This and some demonstration ones remain on the file system but Solr does not use them whatsoever in this mode.
+
+When you create a collection in SolrCloud, you can specify a named configset -- possibly shared.
+If you don't, then the `_default` will be copied and given a unique name for use by this collection.
+
+A configset can be uploaded to ZooKeeper either via the <<configsets-api.adoc#configsets-api,Configsets API>> or more directly via <<solr-control-script-reference.adoc#upload-a-configuration-set,`bin/solr zk upload`>>.
```

Comment:
Think it is bin/solr zk upconfig

Reply:
Good catch; I'm glad you are reading everything.  This error was copied from elsewhere; I fixed it.